swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -63,50 +52,50 @@ Follow the instructions given in this section to develop the service.
         "fee": 7000
     }
     ```
+   
+    ![Define a record](/learn/images/integration-tutorials/sending-a-message-to-a-service/define_record.gif)
 
-    1. Copy the sample JSON payload.
-    2. Open the VS Code [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette).
-    3. Search and select the `Ballerina: Paste JSON as record` command.
-
-    ![Paste JSON as record](/learn/images/integration-tutorials/sending-a-message-to-a-service/paste_json_as_record.gif)
+    > **Note:**
+    > While it is possible to work with the JSON payload directly, using record types offers several advantages including enhanced type safety, data validation, and better tooling experience (e.g., completion).
 
-5. Define the [HTTP service (REST API)](https://ballerina.io/learn/by-example/#rest-service) that has the resource that accepts user requests, retrieves relevant details from the backend service, and responds to the request.
+4. Define the [HTTP service (REST API)](https://ballerina.io/learn/by-example/#rest-service) that has the resource that accepts user requests, retrieves relevant details from the backend service, and responds to the request.
 
-    - Open [Ballerina HTTP API Designer](https://wso2.com/ballerina/vscode/docs/design-the-services/http-api-designer) in VS Code
+    - Open the [Ballerina HTTP API Designer](https://wso2.com/ballerina/vscode/docs/design-the-services/http-api-designer) in VS Code
    
     - Use `/healthcare` as the service path (or the context) of the service, which is attached to the listener listening on port `port`.
```

Comment:
Needs to be updated, no longer have `port`.

Reply:
Oh Missed it. Thanks!